<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LC_DlgQuickSelection</class>
 <widget class="QDialog" name="LC_DlgQuickSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QComboBox" name="cbEntityType">
     <property name="toolTip">
      <string>Type of the entity to match</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QCheckBox" name="cbAppendToCurrentSelectionSet">
     <property name="toolTip">
      <string>If selected, matched entities will be added to the current selection. Otherwise, current selection will be replaced by new one.</string>
     </property>
     <property name="text">
      <string>Append to current selection set</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>How to Apply</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="rbIncludeInSelectionSet">
        <property name="toolTip">
         <string>If checked, the object with matched value will be included into new selection set.</string>
        </property>
        <property name="text">
         <string>Include in new selection set</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="rbExcludeFromSelectionSet">
        <property name="toolTip">
         <string>If checked, object that does not match the condition will be included into selection set.</string>
        </property>
        <property name="text">
         <string>Exclude from new selection set</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cbApplyTo">
     <property name="toolTip">
      <string>Defines to which objects selection conditions should be applied to</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="cbValue">
     <property name="toolTip">
      <string>Value to match</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="cbOperator">
     <property name="toolTip">
      <string>Logical operation applied to the property's value</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Entity Type:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Apply To:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Value:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Properties:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Operator:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" rowspan="2">
    <widget class="QListWidget" name="lvProperties"/>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QToolButton" name="tbManualSelect">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../../../res/icons/icons.qrc">
       <normaloff>:/icons/select.lci</normaloff>:/icons/select.lci</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../../res/icons/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LC_DlgQuickSelection</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LC_DlgQuickSelection</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
